ISISComputingGroup/EPICS-motorExtensions
# Ticket 4112: NIMROD jaws manager

See https://github.com/ISISComputingGroup/IBEX/issues/4112

To test:
* Run the system test in https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/pull/195 against master. You will need to consider the following to make them work:
     - Polaris' settings will need to be updated as in this ticket
     - The PV for GEM's sample jaws has changed (which is dealt with in an upgrade script). You should still be able to run the tests by just changing the PV
* Run the same tests against this branch
* Confirm they pass

Documentation has been expanded for the generic case at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Jaws-Managers and for individual beamlines linked from that ticket.

See https://github.com/ISISComputingGroup/EPICS-upgrade/commit/e18878be65a36a6b4ac84131f3419399978797cc for changes to GEM PV. 

Given the change in settings would only effect a single instrument it seemed overkill to add to the upgrade script so added note at https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev#update-information

- [ ] If you have updated the `settings` directory, have you added a corresponding step to the upgrade script to update affected instruments?
